index.html

```diff
@@ -67,122 +85,125 @@ <h1>Hey there. I’m Mamun</h1>
+                    <button type="button" class="c-btn">See Project</button>
                 </div>
-                <button type="button" class="c-btn">See Project</button>
-            </div>
-        </article>
-        <article class="c-posts__item card-wrapper">
-            <div class="card-image-wrapper"></div>
-            <div class="card-content-wrapper">
-                <h2 class="work-card-title">Multi-Post Stories Gain + Glorys</h2>
-                <div class="tech-tags">
-                    <ul>
-                        <li><a href="#">Ruby on rails</a></li>
-                        <li><a href="#">css</a></li>
-                        <li><a href="#">JavaScript</a></li>
-                        <li><a href="#">html</a></li>
-                    </ul>
+            </article>
+            <article class="c-posts__item card-wrapper">
+                <div class="card-image-wrapper"></div>
+                <div class="card-content-wrapper">
+                    <h2 class="work-card-title">Multi-Post Stories Gain + Glorys</h2>
+                    <div class="tech-tags">
+                        <ul>
+                            <li><a href="#">Ruby on rails</a></li>
+                            <li><a href="#">css</a></li>
+                            <li><a href="#">JavaScript</a></li>
+                            <li><a href="#">html</a></li>
+                        </ul>
+                    </div>
+                    <button type="button" class="c-btn">See Project</button>
                 </div>
-                <button type="button" class="c-btn">See Project</button>
-            </div>
-        </article>
+            </article>
+        </div>
     </section>
 
     <section id="about-me" class="c-posts">
-        <article class="c-posts__item">
-            <h2 class="section-title mb-0">About me</h2>
-            <p class="about-text">
-                I can help you build a product , feature or website Look through some of my work and experience! If you
-                like what you see and have a project you need coded,
-                don’t hestiate to contact me.
-            </p>
-            <button type="button" class="c-btn">Get My Resume</button>
-        </article>
-        <article class="c-posts__item card-wrapper skill-main-wrapper">
-            <div class="skill-image-wrapper">
-                <div class="">
+        <div class="container">
+            <article class="c-posts__item">
+                <h2 class="section-title mb-0">About me</h2>
+                <p class="about-text">
+                    I can help you build a product , feature or website Look through some of my work and experience! If
+                    you
+                    like what you see and have a project you need coded,
+                    don’t hestiate to contact me.
+                </p>
+                <button type="button" class="c-btn">Get My Resume</button>
+            </article>
+        </div>
```

Comment:
- [ ] Kindly consider placing the background image here in this about me section as seen in the Figma design because we need to stick to the figma design.
your design
![dfgh](https://user-images.githubusercontent.com/96683843/214651998-0a3f7412-f484-468f-807c-8a898a65ffb9.jpg)

figma design
![fdsa](https://user-images.githubusercontent.com/96683843/214651963-1a32898e-4c66-4f33-b752-ef2a9af39bc3.jpg)

Reply:
@Zuhaib042 in figma those are 2 individual image , 1 is big and another is small does not know how to fit this.